### DISCUSSION TITLE
Define a "Getting Started" page for OKD/OpenShift beginners

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [okd-project/okd#1286] [Ideas]
Let's use this thread to hash out what a "Getting Started" page for OKD/OpenShift beginners would need.

**Comment by brianinnes** [comment]
This item being tracked on the OKD planning board : https://github.com/okd-project/planning/issues/5

**Comment by PaulCapestany** [comment]
As a newbie to okd, more verbose clarity on the differences and tradeoffs between okd and OpenShift/Red Hat's paid offerings would be great (e.g. as I mentioned in this [#openshift-users Slack thread](https://kubernetes.slack.com/archives/C6AD6JM17/p1664394025886929) I was a bit confused by okd's built-in OperatorHub stating that certain Red Hat operators needed subscriptions, and also okd docs that mention that certain nodes needed to be paid for)... imo part of "Getting Started" is understanding potential costs, pros vs cons of using okd vs something else, etc.
